MIGRATION.md

```diff
@@ -0,0 +1,130 @@
+# Upgrading from 4.x to 5.x
+
+In this version upgrade there are a few breaking changes, this guide should help you to make necessary code changes.
+
+## Integrations
+
+We moved optional integrations into their own package called `@sentry/integrations`. Also we made a few default
+integrations now optional. This is probably the biggest breaking change regarding the upgrade.
+
+Integrations that are now opt-in and were default before:
+
+- Dedupe (responsible for sending the same error only once)
+- ExtraErrorData (responsible for doing fancy magic trying to extract data out of the error object)
+
+Integrations that were pluggable/optional before also live in this package:
+
+- Angular (browser)
+- Debug (browser/node)
+- Ember (browser)
+- ReportingObserver (browser)
+- RewriteFrames (browser/node)
+- Transaction (browser/node)
+- Vue (browser)
+
+### How to use `@sentry/integrations`?
+
+Lets start with the approach if you install `@sentry/browser` / `@sentry/electron` with `npm` or `yarn`.
+
+Given you have a `Vue` application running, in order to use the `Vue` integration you need to do the following:
+
+With `4.x`:
+
+```js
+import * as Sentry from '@sentry/browser';
+
+Sentry.init({
+  dsn: '___PUBLIC_DSN___',
+  integrations: [
+    new Sentry.Integrations.Vue({
+      Vue,
+      attachProps: true,
+    }),
+  ],
+});
+```
+
+With `5.x` you need to install `@sentry/integrations` and change the import.
+
+```js
+import * as Sentry from '@sentry/browser';
+import * as Integrations from '@sentry/integrations';
```

Comment:
I tried 5.0.0-rc.3:

```
> require('@sentry/integrations')
{ Error: Cannot find module '@sentry/integrations'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
    at Function.Module._load (internal/modules/cjs/loader.js:506:25)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at require (internal/modules/cjs/helpers.js:20:18) code: 'MODULE_NOT_FOUND' }
```

Reply:
@wong2 Yeah, I changed it in this PR to expose `index.js` in package.json
This will work with `rc.4` sorry

Reply:
@HazAT oh I missed that, thanks a lot